### DISCUSSION TITLE
minified/compacted output

### DISCUSSION THREAD

**Discussion by niXman** [danielaparker/jsoncons#399] [General] [answered]
hi,

could anybody tell me is there any way to get minified/compacted output of json in text?

I tried to figure it out myself by tuning `jsoncons::json_options`, but unsuccessful...

best!

**Comment by danielaparker** [comment, marked as answer]
What have you tried? You should see lots of examples of compact output, e.g.  [here](https://github.com/danielaparker/jsoncons/blob/master/doc/Examples.md#encode-1) and [here](https://github.com/danielaparker/jsoncons/blob/master/doc/Examples.md#write-some-json-push)

**Comment by niXman** [comment]
> What have you tried?

im tried to set `indent_size` and `line_length_limit` to zero.
yes, that works.

thanks!
